app/assets/stylesheets/views/_topics.scss

```diff
@@ -123,7 +123,7 @@
   .browse-container {
     background-color: $page-colour;
     min-height: 500px;
-    padding: 0 30px 45px 30px;
+    padding: 0 0 45px 0;
```

Comment:
I guess the way I see it is it's the difference between.

`setPadding(0, 0, 45px, 0)` and `setPadding({
  top: 0,
  left: 0,
  bottom: 45px,
  right: 0
})`

```css
padding: 0;
padding-bottom: 45px;
```

vs

```css
padding: 0 0 45px 0;
```

If you can write the explicit long hand it makes it much easier to know what's happening without having to remember the ordering.

Not worth pushing for this PR though, wider discussion. 👍